src/pybind/mgr/volumes/fs_subvolume.py

```diff
@@ -0,0 +1,284 @@
+"""
```

Comment:
This file really looks like a general purpose thing that is not necessarily part of the mgr/volumes module. 

Is this file useful for other users of the Ceph's python client libraries? Does it make sense to put it in a place where it can be used by others?

Reply:
We want the mgr to be the SSOT and arbiter of volumes; so we don't want other users.

Reply:
ok!